src/data/petsData.ts

```diff
@@ -42,6 +43,7 @@ export const petsData: PetProps[] = [
   },
   {
     name: "dżeki",
```

Comment:
@Delicadency You need to use the name without the Polish sign, possibly change the routing, with dżeki = dzeki because with the Polish sign routing does not work!